extension/content.css

```diff
@@ -10,61 +11,65 @@ mark {
 
 /* SHADOW, HIGHLIGHTER, and KEYWORD CSS */
 .sentenceHighlighterBlueShadowBlue {
-	background-color:rgb(230, 239, 253); /* Light Blue highlighter */
+	/* TODO: Integrate different color scheme. Keeping values here for near-term future use */
+	background-color:rgb(230, 239, 253) !important; /* Light Blue highlighter */
```

Comment:
Why the !important ?
IIRC it prevents the fading animation from working, but not sure here.

Reply:
Mark causes yellow background, so have all mark classes (which is now every sentence) inherit background !important, so need this to override the mark !important. (See comment in code). Long term solution is to find where the mark defaults are, and nuke them